### PR TITLE
[REF][account_tax_cash_basis_fix_currency_rate_difference] - Module w…

### PR DESCRIPTION
**Pull request by ougc27**
@alan196  Could you review this PR?

### CODE REVIEW

File: account_tax_cash_basis_fix_currency_rate_difference/models/account_partial_reconcile.py [hunk 1]
```diff
@@ -4,151 +4,85 @@
 
 from __future__ import division
 
-from odoo import _, models
+from openerp import _, models
```
Comment (alan196) [inline account_tax_cash_basis_fix_currency_rate_difference/models/account_partial_reconcile.py]: From odoo

**Review by alan196 — CHANGES_REQUESTED** [review]

File: account_tax_cash_basis_fix_currency_rate_difference/models/account_partial_reconcile.py (hunk 1 above)
Comment (ougc27) [reply account_tax_cash_basis_fix_currency_rate_difference/models/account_partial_reconcile.py]: @alan196 changes are done!

**Comment by alan196** [comment]
What does this PR do or fix??...

**Comment by ougc27** [comment]
This PR deletes the analytic accounting behavior in the tax cash basis difference of the currency exchange difference

**Review by alan196 — APPROVED** [review]

**Pull request merged**